src/navigation/screens/app/CartScreen.js

```diff
@@ -1,21 +1,90 @@
-import { Text, View } from "react-native-ui-lib";
+import { View } from "react-native-ui-lib";
 import { ShoppingListContext } from "../../../constants";
-import { useContext } from "react";
+import { useContext, useState } from "react";
+import { KContainer, KSpacer } from "../../../components";
+import {
+  Alert,
+  KeyboardAvoidingView,
+  ScrollView,
+  TextInput,
+  TouchableOpacity,
+} from "react-native";
+import { Colors, Typographies } from "../../../constants";
+import AntDesign from "react-native-vector-icons/AntDesign";
+import { KQuantity } from "../../../components/KQuantity";
 
 export const CartScreen = () => {
-  //TODO: PENTRU ANDRA: Acesta este contextul
   const { shopList, setShopList } = useContext(ShoppingListContext);
 
-  //TODO:  De forma asta este contextul, doar ca initial e gol
-  const auxShopList = [
-    { name: "Egg", amount: 1 },
-    { name: "Milk", amount: 15 },
-    { name: "Bred", amount: 4 },
-  ];
+  const [addIngredient, setAddIngredient] = useState("");
 
   return (
-    <View flex center>
-      <Text>Cart page</Text>
-    </View>
+    <KContainer scrollable={false}>
```

Comment:
KContainer should be scrollable and you should remove the View and Scroll on lines 25 and 26. To achieve the text input stuck on the bottom you could use something like this:
```
<>
   <KContainer> ... </KConatiner>
   <View> ... </View>
</>
```